x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -4,38 +4,42 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
 import { EuiButtonEmpty } from '@elastic/eui';
-import PropTypes from 'prop-types';
+import { capitalize, get, isEmpty } from 'lodash';
+import React from 'react';
 import styled from 'styled-components';
+import { Stackframe } from '../../../../../typings/APMDoc';
+import { Error } from '../../../../../typings/Error';
+import { Context } from '../../../../../typings/Transaction';
```

Comment:
It feels wrong to get `Context` from `Transaction` since you are using it for Error. `Error` should have its own `Context` - you can access it like `Error['context']`.